.eslintrc

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@vhx/eslint-config-vhx"
+  "extends": "@vhx/eslint-config-vhx",
+  "rules": {
+    "jsx-a11y/href-no-hash": 0
```

Comment:
Once the latest version of our eslint config is published, we can remove this one-off rule.